### DISCUSSION TITLE
图片格式转换工具不可用

### DISCUSSION THREAD

**Discussion by ghost** [he3-app/he3-feedback#285] [Bug Report]
(.webp) 转 其他

**Comment by lyzhang1999** [comment]
我这边测试是 webp 转 jpg, png, bmp 是正常的，可以把图片贴上来吗？
![image](https://github.com/he3-app/he3-feedback/assets/8142348/4ceabb5b-e14d-41a5-a6dc-f98b4e44ce52)

这是我使用的 webp 图片：
[1.sm.webp.zip](https://github.com/he3-app/he3-feedback/files/12839686/1.sm.webp.zip)

**Comment by ghost** [comment]
我是在windows 上面测试的。
https://doc.traefik.io/traefik/assets/img/middleware/overview.png
![image](https://github.com/he3-app/he3-feedback/assets/71524776/4cac96ec-c91d-4aee-856f-e3e32f61f9a7)
